src/controllers/GraphController.ts

```diff
@@ -660,8 +661,15 @@ export interface IGraphEdgeDataPoint {
   target: number | string;
 }
 
+export interface EdgePrefixedOptions {
```

Comment:
can you move that next to the `IEdgeLineOptions` and ensure that both cover the same properties?

Reply:
or even better based on https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-1.html#key-remapping-in-mapped-types it might pe possible to just derive this type without listing every property